docs/en/Configuring-Alluxio-with-S3.md

```diff
@@ -75,42 +62,35 @@ Alternatively, these configuration settings can be set in the `conf/alluxio-env.
 details about setting configuration parameters can be found in
 [Configuration Settings](Configuration-Settings.html#environment-variables).
 
+After these changes, Alluxio should be configured to work with S3 as its under storage system, and
+you can try [Running Alluxio Locally with S3](#running-alluxio-locally-with-s3).
+
 ### Nested Mount
 An S3 location can be mounted at a nested directory in the Alluxio namespace to have unified access
 to multiple under storage systems. Alluxio's [Command Line Interface](Command-Line-Interface.html) can be used for this purpose.
 
-```bash 
+```bash
 $ ./bin/alluxio fs mount --option aws.accessKeyId=<AWS_ACCESS_KEY_ID> --option aws.secretKey=<AWS_SECRET_KEY_ID>\
   /mnt/s3 s3a://<S3_BUCKET>/<S3_DIRECTORY>
 ```
 
 ### Enabling Server Side Encryption
 
-If you are using s3a, you may encrypt your data stored in S3. The encryption is only valid for data
-at rest in s3 and will be transferred in decrypted form when read by clients.
+You may encrypt your data stored in S3. The encryption is only valid for data at rest in s3 and will
```

Comment:
nit: we should be consistent about whether S3 is capitalized or not

Reply:
Updated, thanks.